tests/main/views/test_direct_award.py

```diff
@@ -672,6 +672,26 @@ def test_award_contract_we_are_still_assessing_redirects_on_post(self, choice, e
         assert res.status_code == 302
         assert res.location.endswith(expected_redirect)
 
+    def test_we_are_still_assessing_calls_api(self):
+        self.login_as_buyer()
+
+        self.client.post('/buyers/direct-award/g-cloud/projects/1/did-you-award-contract',
+                         data={'did_you_award_a_contract': 'still-assessing'})
+
+        self.data_api_client.mark_direct_award_project_as_still_assessing.assert_called_once()
+
+    def test_still_assessing_flashes(self):
+        self.login_as_buyer()
+
+        self.client.post('/buyers/direct-award/g-cloud/projects/1/did-you-award-contract',
+                         data={'did_you_award_a_contract': 'still-assessing'})
+
+        self.assert_flashes(
+            "Your response for ‘My procurement project &lt;’ has been saved. "
```

Comment:
A project name - or any other user input - can contain characters that might be significant to HTML, and so have to be escaped. We don't expect the API to prevent users from using angle brackets (etc) in their names. We expect the frontends to handle strings safely, i.e. know which strings are safe and which ones aren't.

Jinja and friends tend to handle this right (magic is in the `escape` function) - but not if (as was the case here) if we are combining user input with custom markup (a `<br>` tag as it happened, see the diff for my last commit).

Reply:
Ah - so if we have got rid of the `<br>`, do we still need this?

Reply:
As discussed - but for posterity: yeah it's good to be checking we're doing the right thing, as introducing new `Markup` instances with XSS vulns is *really* easy to do.